core/args.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""
+Functions to parse and retrieve command line arguments.
+"""
+
+import os
+import sys
+import argparse
+
+nbuild_args = None
+nbuild_parser = None
+
+
+def parse_args():
+    """
+    Parses the command line arguments.
+    This function may exit if the command line arguments are invalid.
+    """
+    global nbuild_parser
+    global nbuild_args
+
+    nbuild_parser = argparse.ArgumentParser(
+        description='Compiles packages from a build Manifest.'
+    )
+    nbuild_parser.add_argument(
+        '-o',
+        '--output-dir',
+        default=os.path.join(  # Default path is script_dir/packages
+            os.getcwd(),
+            os.path.dirname(sys.argv[0]),
+            'packages',
+        ),
+        help='Output directory for built packages. Default: packages/',
```

Comment:
Shouldn't those help messages be between double quotes instead of single ones?

Reply:
You're right :+1: